debug-dump/README.md

```diff
@@ -0,0 +1,4 @@
+# debug-dump
+
+This is a library providing simple functionality like that found in
+`Debug.Trace`, but it dumps output to a file rather than the terminal.
```

Comment:
Why not just add these functions to `PlutusPrelude`?

Reply:
Mainly because I might want to reuse this library. I can expose these functions in `PlutusPrelude` if you'd like.

Reply:
I'd prefer to export them from `PlutusPrelude` or to make a completely unrelated package, put it on `hackage` and fetch it from there.

Reply:
Fair enough. I'll have a look at creating another repo.